src/lib/index/shared/configValidator.js

```diff
@@ -35,6 +35,9 @@ const
 		if (config.url == null || !_.isString(config.url)) {
 			throw new Error('Invalid parameter: url not a string.');
 		}
+		if (!config.return_buffers) {
```

Comment:
Why have the property at all, if it always errors when the property is falsey?

Reply:
It's a Redis setting. I'll move the setting of the config option to the point that we call into Redis and remove from our validation.